httpbin/helpers.go

```diff
@@ -50,6 +50,11 @@ func getClientIP(r *http.Request) string {
 		return clientIP
 	}
 
+	// Special case Cloudflare for the value directly.
```

Comment:
Style nit:  The comment above the previous conditional already explains this case, let's drop this comment and delete the blank line between the fly and cloudflare conditionals.